src/controller/python/chip-device-ctrl.py

```diff
@@ -849,6 +849,13 @@ def qr_code_parse(qr_code):
     except Exception as e:
          return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
+def pin_code_code_parse(pin_code):
+    try:
+        result = SetupPayload().ParseManualPairingCode(pin_code).Dictionary()
+        return __get_response_dict(status = StatusCodeEnum.SUCCESS, result = result)
+    except Exception as e:
+         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
```

Comment:
remove spaces in keyword argument when passing to function like: 

`__get_response_dict(status=StatusCodeEnum.SUCCESS, result=result)`

Only use spaces when assigning default values to arguments when first defining a method or class.